clusterloader2/pkg/measurement/util/pods_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestConvertPodsStartupStatusToString(t *testing.T) {
```

Comment:
While +1 to adding some tests, It's not clear if the string format is worth testing. It looks more like "change-detector" to me, rather than valuable test.

What kind of regression can we prevent with this?

Reply:
I wanted to confirm that providing `<nil>` error does not break formating.

Reply:
Actually, removed this part. It was too noisy - even for successful 100 node run from presubmit hundred of additional log lines were added.

Moved log to only show single error in case of timeout.